yarn.lock

```diff
@@ -2023,6 +2023,20 @@
     "@smithy/types" "^2.3.3"
     tslib "^2.5.0"
 
+"@aws-sdk/middleware-sdk-ec2@3.433.0":
```

Comment:
Can this be `3.421.0`?

Reply:
Ran `yarn install` locally and pushed the changes. This is fixed